### PR TITLE
chart: update appVersion in chart updates

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: circle/docker-release-image.sh [hunk 1]
```diff
@@ -285,6 +285,24 @@ chart_update_image() {
   esac
 }
 
+chart_update_appVersion() {
```
Comment (sameersbn) [inline circle/docker-release-image.sh]: This function we determine the appVersion from the image tag. I'm not sure how we should deal with charts that consist two or more images, fe. the [parse chart](https://github.com/bitnami/charts/blob/master/incubator/parse/values.yaml) consists of parseServer and parseDashboard images, each having a different version, and updating each of these will trigger a separate chart update. 

Suggestion are welcome /cc @prydonius

Comment (prydonius) [inline circle/docker-release-image.sh]: Perhaps the simplest way is to just use an env var to skip the appVersion update in some repos. So, we could disable this in [parse-dashboard](https://github.com/bitnami/bitnami-docker-parse-dashboard) but enable it in [parse](https://github.com/bitnami/bitnami-docker-parse), that way we're only following the parse appVersion. The downside is that it's difficult to get visibility into what appVersion is being followed and it is easy to configure incorrectly and have multiple components updating the appVersion.

Comment (sameersbn) [inline circle/docker-release-image.sh]: Thanks for the suggestion. I will add a ci script variable to disable the `appVersion` updates. It should work fine for our current use-cases.

**Comment by sameersbn** [comment]
@prydonius added `SKIP_CHART_APP_VERSION_UPDATE` ci variable to disable `appVersion` updates.

**Pull request merged**